packages/cli/commands/sandbox/create.js

```diff
@@ -27,26 +47,221 @@ exports.handler = async options => {
   const accountConfig = getAccountConfig(accountId);
   const env = getValidEnv(getEnv(accountId));
 
+  trackCommandUsage('sandbox-create', null, accountId);
+
+  // Default account is not a production portal
+  if (
+    accountConfig.sandboxAccountType &&
+    accountConfig.sandboxAccountType !== null
+  ) {
+    trackCommandUsage('sandbox-create', { successful: false }, accountId);
+    logger.error(
+      i18n(`${i18nKey}.failure.creatingWithinSandbox`, {
+        sandboxType: getSandboxTypeAsString(accountConfig.sandboxAccountType),
+        sandboxName: accountConfig.name,
+      })
+    );
+    process.exit(EXIT_CODES.ERROR);
+  }
+
+  let typePrompt;
+  let namePrompt;
+
+  if ((type && !sandboxTypeMap[type]) || !type) {
+    if (!force) {
```

Comment:
Having a little trouble grasping this: What happens if `force` is true?

Reply:
If a user were to run this command with the force option, we require both the type and a name for creation. If one or none of those are provided, we throw an error saying they must provide those options to proceed

Reply:
Would it be better to have defaults here? like if no type is provided, have a default selected and prefill a name?